lib/diagnostic.rb

```diff
@@ -115,7 +118,7 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = person_hash = {age: "", first_name: ""}
 
 # Set a default return value to your `person` hash that concatenates the value
 # of person[:first_name] and a message about them.
```

Comment:
Answer to the last question: `Diagnostic.hash_default = 'person.default = "#{person[:first_name]} is great"`